huaweicloud/endpoints.go

```diff
@@ -100,6 +100,11 @@ var allServiceCatalog = map[string]ServiceCatalog{
 		Name:    "functiongraph",
 		Version: "v2",
 	},
+	"swrv2": {
```

Comment:
suggest to use `swr`, then we can export it in `provider/endpoints`

Reply:
done